### DISCUSSION TITLE
Adding Ascendo Speakers

### DISCUSSION THREAD

**Discussion by sk9592** [pierreaubert/spinorama#273] [General]
I noticed that you recently added a ton of new speakers to the website, including the Kanto and HSU speakers I requested. Thanks for that!

The next time you update the site, do you think you would be able to incorporate some of the measurements on Ascendo's speakers? They post their Klippel graphs on their website. I don't know whether the data quality will be to your standard. They apply 1/6 smoothing to everything. 

These are the measurements for their 6" speakers:

The6 Passive Narrow: https://www.aia-cinema.com/products/2023_Products/The_6_PASSIVE_NARROW/THE6_PASSIVE_NARROW_-_Measurements_Report.pdf

The6 Passive On Wall: https://www.aia-cinema.com/products/2023_Products/THE_6_PASSIVE_ON_WALL/THE6_PASSIVE_ON_WALL_-_Measurements_Report.pdf

The6 Pro Passive Ceiling: https://www.aia-cinema.com/products/2023_Products/THE_6_PRO_PASSIVE_CEILING/THE6_PRO_PASSIVE_CEILING-_Measurements_Report.pdf

The6 Passive Wedge30: https://www.aia-cinema.com/products/Passive/The_6_PASSIVE_WEDGE_30/THE6_PASSIVE_WEDGE_30_-_Measurements_Report.pdf

They also have measurements for their 10", 12", and 15" speakers. So there are about 20 speakers total. If you're not willing to add that many, that's fine.

**Comment by pierreaubert** [comment]
I contacted them last year and chatted with them. They didnt send the data. Their speakers are great btw I listened to them a few times and that's good stuff.

If you are willing to digitalized the plots with webplotdigitizer I will do the rest.
